.travis.yml

```diff
@@ -5,6 +5,6 @@ node_js:
   - "8"
   - "6"
   - "4"
-script: "npm run jshint && npm run test-cover"
+script: "ln -s .. node_modules/sharedb; npm run jshint && npm run test-cover"
```

Comment:
Dropping another note for the future, this is to work around the circular dependency issue between sharedb and sharedb-mingo-memory:
https://github.com/share/sharedb/pull/226#issuecomment-406297957

I'm OK with this workaround for now to unblock the tests as the workaround is isolated to Travis, until we figure out a better solution in the linked issue above.

Reply:
Actually, maybe it's better to get the #226 merged first, as it also updates the Travis config's Node versions to reflect latest + LTE versions.

Reply:
Sure. Happy to wait and rebase. Would just be nice to get this through to unblock other PRs.